Grbl_Esp32/i2s_ioexpander.cpp

```diff
@@ -197,7 +197,7 @@ static int i2s_stop() {
 }
 
 int i2s_ioexpander_start() {
-  i2s_start();
```

Comment:
As a stylistic issue, when you are committing code, it is better to delete lines instead of commenting them out.  Commented-out lines make the code untidy and can make the reader wonder about the intent. The revision history will let you find the old code later if you need it.

Reply:
Since we're using a code management system, I also think we should remove any unnecessary code before committing it.
(I'm embarrassed. I tried it out in the comments and it worked well, so I was excited and committed as is.)